src/jobs/remove_completed.py

```diff
@@ -0,0 +1,87 @@
+"""Removes completed torrents that have specific tags/categories."""
+
+from src.jobs.download_client_removal_job import DownloadClientRemovalJob
+from src.utils.log_setup import logger
+
+COMPLETED_STATES = [
+    "stoppedUP",
+    "pausedUP",  # Older qBittorrent versions
+]
+
+
+class RemoveCompleted(DownloadClientRemovalJob):
+    """Job to remove completed torrents that match specific tags or categories."""
+
+    async def run(self) -> int:
+        if self.download_client_type == "sabnzbd":
```

Comment:
Not sure I understand why you define here that you don't want to process sabnzb, since there's many other clients that you don't want to process either (deluge, transmission, nzbget, etc)
Instead, would it make sense to say             
if not download_client_type == "qbittorrent":?


```suggestion
        if not self.download_client_type == "qbittorrent":
```